scss/tech.scss

```diff
@@ -0,0 +1,47 @@
+.tech-container {
+  align-items: center;
+  background-color: $green;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  height: 350px;
+  width: 260px;
+
+  &::before {
+    border: 2px solid $green;
+    border-radius: 10px;
+    content: '';
+    height: 100%;
+    left: -15px;
+    position: relative;
+    top: -20px;
+    width: 100%;
+  }
+
+  .tech-logo {
+    height: 100px;
+    left: 0;
+    position: relative;
+    top: 10px;
```

Comment:
this should be handled through padding in the parent instead of doing it this way.